### ISSUE TITLE
🐞 Bug Report: "elementor/popup/show" js event not trigger in 3.9 (ED-9041)

### ISSUE THREAD

**Issue by marcatop** [elementor/elementor#20708]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The js event "elementor/popup/show" and maybe others two not triggered with Elementor 3.9, but working when I rollback to 3.8.

### Steps to reproduce

1) Update to 3.9
2) Try to got js event for modal popup

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

Operating System: | Linux |  
-- | -- | --
Software: | Apache/2.4.54 (Unix) OpenSSL/1.1.1f |  
MySQL version: | mariadb.org binary distribution v10.5.13 |  
PHP Version: | 8.1.12 |  
PHP Memory Limit: | 256M |  
PHP Max Input Vars: | 1000 |  
PHP Max Post Size: | 256M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected

**Comment by blickwerker** [comment]
"elementor/popup/hide" broken as well.

**Comment by mnemotiv** [comment]
temporary workaround (backwards compatibility):

```js
window.addEventListener( 'elementor/popup/show', function ( e ) {
	$( document ).trigger( 'elementor/popup/show', [ e.detail.id, e.detail.instance ] );
} );

window.addEventListener( 'elementor/popup/hide', function ( e ) {
	$( document ).trigger( 'elementor/popup/hide', [ e.detail.id, e.detail.instance ] );
} );
```

of course, there's "Elementor way" to work with events, but...

**Comment by nicholaszein** [comment]
Hello everyone, thanks for the report. 🙏

We are aware of this issue, and a fix is on the way! 💪 

Hey @marcatop, 

please, when submitting an issue make sure to follow the guidelines in the submission form. These include:

- **Description:** In your description, you need to explain in detail what happened, as you would explain to someone that doesn't know anything about your website or Elementor. This is important information that will help solve your issue quicker; Be thorough with your description;
- **Steps to replicate:** Think of it as explaining to someone over the phone how to replicate your issue. They can't see the problem, so explain it in detail in a way that we can try to replicate your issue in our own environments;
- **System info:** Please include the [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml) report available in your WordPress Dashboard > Elementor > System Info. This information is crucial for us to create an exact copy of your environment and validate your issue. Watch [this video](https://www.youtube.com/watch?v=tDXdynsnYuA) to learn how to download the report. We can't accept issues without it.

Please update your issue, and provide the System Info as explained above.

Kind regards

**Comment by rami-elementor** [comment]
Fixed in Elementor Pro 3.9.2
